src/Symfony/Component/Serializer/Exception/AggregatedException.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Exception;
+
+class AggregatedException extends RuntimeException implements AggregableExceptionInterface
+{
+    /**
+     * @var array
+     */
+    private $collection;
+
+    public function __construct(\Throwable $previous = null)
+    {
+        parent::__construct('Errors occurred during the denormalization process', 0, $previous);
+    }
+
+    public function has(): bool
```

Comment:
Maybe could this be a property to prevent an extra function call on the hot path?

Reply:
Done